src/com/lftechnology/java/training/srijan/basics/coreapp/ui/EmployeeView.java

```diff
@@ -0,0 +1,71 @@
+package com.lftechnology.java.training.srijan.basics.coreapp.ui;
+
+import com.lftechnology.java.training.srijan.basics.coreapp.controller.EmployeeManagementController;
+import com.lftechnology.java.training.srijan.basics.coreapp.domain.UserRole;
+
+import java.util.Scanner;
+import java.util.logging.Level;
+import java.util.logging.Logger;
+
+public class EmployeeView {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Add a private constructor to hide the implicit public one. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1118)